src/Globals.d.ts

```diff
@@ -0,0 +1 @@
+declare module "*.module.pcss";
```

Comment:
You can use .css files with postcss. It's not necessary, but I think you might haven't to write a declaration for css files.